x-pack/plugins/alerts/server/types.ts

```diff
@@ -122,7 +126,7 @@ export interface RawAlert extends SavedObjectAttributes {
   schedule: SavedObjectAttributes;
   actions: RawAlertAction[];
   params: SavedObjectAttributes;
-  scheduledTaskId?: string;
+  scheduledTaskId?: string | null;
```

Comment:
Seems we *were* setting this as `Null` in some places, but the type system got confused and didn't notice it.
Adding the `updateMeta` call revealed the mistake and forced this addition.